docker-compose.yml

```diff
@@ -74,19 +78,50 @@ services:
     environment:
       - POSTGRES_USER=admin
       - POSTGRES_PASSWORD=admin_password
-      - DATABASE_NAME=court-register-db
-      - DATABASE_ENDPOINT=court-register-db
+      - POSTGRES_DB=court_register_db
+
+  prison-register:
+    image: quay.io/hmpps/prison-register:latest
+    networks:
+      - hmpps
+    container_name: prison-register
+    depends_on:
+      - hmpps-auth
+    ports:
+      - "8103:8080"
```

Comment:
Used same port mapping as https://github.com/ministryofjustice/send-legal-mail-to-prisons/blob/main/docker-compose.yml#L49